Formula/jack.rb

```diff
@@ -45,6 +45,13 @@ def plist; <<~EOS
     <dict>
       <key>Label</key>
       <string>#{plist_name}</string>
+      <key>EnvironmentVariables</key>
+      <dict>
+           <key>TMPDIR</key>
```

Comment:
are you sure it fails without setting this? We're not having to set TMPDIR in any other plist.